acceptence_test/main.tf

```diff
@@ -0,0 +1,378 @@
+terraform {
+  required_version = " >= 0.11.0"
+}
+
+
```

Comment:
Please also put logs of backward compatibility tests of this plugin with terraform 0.11.

Reply:
Terraform 0.11 test Results.

Test results using terraform 0.11

$ export TF_VAR_ULTRADNS_USERNAME=XXXXXXXXXXX
$ export TF_VAR_ULTRADNS_PASSWORD=XXXXXXXXXXX
$ export TF_VAR_ULTRADNS_BASEURL="https://restapi.test.ultradns.net:443"
$ terraform --version
Terraform v0.11.0

Your version of Terraform is out of date! The latest version
is 0.12.24. You can update by downloading from www.terraform.io/downloads.html


$ make testacc
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test  acceptence_test/acceptence_test.go  -v  -timeout 120m
=== RUN   TestUltraDNSPlugin
=== PAUSE TestUltraDNSPlugin
=== CONT  TestUltraDNSPlugin
TestUltraDNSPlugin 2020-04-27T16:47:07+05:30 retry.go:72: terraform [init -upgrade=false -no-color]
TestUltraDNSPlugin 2020-04-27T16:47:07+05:30 command.go:87: Running command terraform with args [init -upgrade=false -no-color]
TestUltraDNSPlugin 2020-04-27T16:47:07+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:47:07+05:30 command.go:158: Initializing provider plugins...
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: Terraform has been successfully initialized!
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: You may now begin working with Terraform. Try running "terraform plan" to see
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: any changes that are required for your infrastructure. All Terraform commands
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: should now work.
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: If you ever set or change modules or backend configuration for Terraform,
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: rerun this command to reinitialize your working directory. If you forget, other
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: commands will detect it and remind you to do so if necessary.
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 retry.go:72: terraform [get -update -no-color]
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:87: Running command terraform with args [get -update -no-color]
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 retry.go:72: terraform [apply -input=false -auto-approve -lock=false -no-color]
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:87: Running command terraform with args [apply -input=false -auto-approve -lock=false -no-color]
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-maximal: Creating...
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   act_on_probes:                   "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   backup_record_rdata:             "" => "10.2.1.3"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   description:                     "" => "traffic controller pool with probes"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   hostname:                        "" => "<computed>"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   max_to_lb:                       "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   name:                            "" => "testprobehttpmaximal.com"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.#:                         "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1576859844.failover_delay: "" => "0"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1576859844.host:           "" => "10.2.1.1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1576859844.priority:       "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1576859844.run_probes:     "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1576859844.state:          "" => "NORMAL"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1576859844.threshold:      "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1576859844.weight:         "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3304434046.failover_delay: "" => "0"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3304434046.host:           "" => "10.2.1.2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3304434046.priority:       "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3304434046.run_probes:     "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3304434046.state:          "" => "NORMAL"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3304434046.threshold:      "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3304434046.weight:         "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   run_probes:                      "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   ttl:                             "" => "30"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   zone:                            "" => "1keufrf3.test"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: ultradns_record.foobar: Creating...
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   hostname:         "" => "<computed>"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   name:             "" => "terraform"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.#:          "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3225190724: "" => "192.168.0.11"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   ttl:              "" => "3600"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   type:             "" => "A"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   zone:             "" => "1keufrf3.test."
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: ultradns_dirpool.test-dirpool-maximal: Creating...
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   conflict_resolve:                                  "" => "GEO"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   description:                                       "" => "Description of pool"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   hostname:                                          "" => "<computed>"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   name:                                              "" => "testdirpoolmaximal.com"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.#:                                     "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.all_non_configured:                  "" => "false"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.geo_info.#:                          "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.geo_info.0.codes.#:                  "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.geo_info.0.codes.540627860:          "" => "Z4"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.geo_info.0.is_account_level:         "" => "false"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.geo_info.0.name:                     "" => "nrGeo"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.ip_info.#:                           "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.ip_info.0.ips.#:                     "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.ip_info.0.ips.820020534.address:     "" => "197.231.41.3"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.ip_info.0.ips.820020534.cidr:        "" => ""
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.ip_info.0.ips.820020534.end:         "" => ""
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.ip_info.0.ips.820020534.start:       "" => ""
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.ip_info.0.is_account_level:          "" => "false"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   no_response.0.ip_info.0.name:                      "" => "nrIP"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.#:                                           "" => "3"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2203440046.all_non_configured:               "" => "false"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2203440046.geo_info.#:                       "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2203440046.geo_info.0.codes.#:               "" => "3"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2203440046.geo_info.0.codes.1225984395:      "" => "US-OK"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2203440046.geo_info.0.codes.2616230423:      "" => "US-MA"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2203440046.geo_info.0.codes.2755418226:      "" => "US-DC"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2203440046.geo_info.0.is_account_level:      "" => "false"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2203440046.geo_info.0.name:                  "" => "North America"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2203440046.host:                             "" => "10.1.1.2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2203440046.ip_info.#:                        "" => "0"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.all_non_configured:               "" => "false"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.geo_info.#:                       "" => "0"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.host:                             "" => "10.1.1.3"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.#:                        "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.ips.#:                  "" => "3"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.ips.2496421098.address: "" => "50.60.70.80"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.ips.2496421098.cidr:    "" => ""
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.ips.2496421098.end:     "" => ""
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.ips.2496421098.start:   "" => ""
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.ips.3125116513.address: "" => ""
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.ips.3125116513.cidr:    "" => "20.20.20.0/24"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.ips.3125116513.end:     "" => ""
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.ips.3125116513.start:   "" => ""
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.ips.3197355849.address: "" => ""
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.ips.3197355849.cidr:    "" => ""
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.ips.3197355849.end:     "" => "200.20.0.10"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.ips.3197355849.start:   "" => "200.20.0.1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.is_account_level:       "" => "false"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4099072824.ip_info.0.name:                   "" => "some Ips"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.442270228.all_non_configured:                "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.442270228.geo_info.#:                        "" => "0"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.442270228.host:                              "" => "10.1.1.1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.442270228.ip_info.#:                         "" => "0"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   ttl:                                               "" => "300"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   type:                                              "" => "A"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   zone:                                              "" => "1keufrf3.test"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: ultradns_rdpool.it: Creating...
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   description:      "" => "Minimal RD Pool"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   hostname:         "" => "<computed>"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   name:             "" => "test-rdpool-minimal"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   order:            "" => "ROUND_ROBIN"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.#:          "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2847814707: "" => "10.6.0.1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   ttl:              "" => "300"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   zone:             "" => "1keufrf3.test"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: ultradns_dirpool.test-dirpool-minimal: Creating...
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   conflict_resolve:                   "" => "GEO"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   description:                        "" => "Minimal directional pool"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   hostname:                           "" => "<computed>"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   name:                               "" => "testdirpoolminimal.com"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.#:                            "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.463398947.all_non_configured: "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.463398947.geo_info.#:         "" => "0"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.463398947.host:               "" => "10.1.0.1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.463398947.ip_info.#:          "" => "0"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   ttl:                                "" => "300"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   type:                               "" => "A"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   zone:                               "" => "1keufrf3.test"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: ultradns_record.it: Creating...
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   hostname:         "" => "<computed>"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   name:             "" => "test-record-txt"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.#:          "" => "4"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.126343430:  "" => "complex answer \\ \""
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1447448707: "" => "simple answer"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3135730072: "" => "quote answer \""
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3337444205: "" => "backslash answer \\"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   ttl:              "" => "3600"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   type:             "" => "TXT"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   zone:             "" => "1keufrf3.test"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-minimal: Creating...
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   act_on_probes:                   "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   backup_record_rdata:             "" => "10.2.0.3"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   description:                     "" => "traffic controller pool with probes"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   hostname:                        "" => "<computed>"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   max_to_lb:                       "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   name:                            "" => "testprobehttpminimal.com"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.#:                         "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1547604723.failover_delay: "" => "0"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1547604723.host:           "" => "10.2.0.1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1547604723.priority:       "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1547604723.run_probes:     "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1547604723.state:          "" => "NORMAL"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1547604723.threshold:      "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1547604723.weight:         "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3308766025.failover_delay: "" => "0"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3308766025.host:           "" => "10.2.0.2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3308766025.priority:       "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3308766025.run_probes:     "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3308766025.state:          "" => "NORMAL"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3308766025.threshold:      "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.3308766025.weight:         "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   run_probes:                      "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   ttl:                             "" => "30"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   zone:                            "" => "1keufrf3.test"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: ultradns_tcpool.test-probe-ping-pool: Creating...
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   act_on_probes:                   "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   backup_record_rdata:             "" => "10.3.0.3"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   description:                     "" => "traffic controller pool with probes"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   hostname:                        "" => "<computed>"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   max_to_lb:                       "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   name:                            "" => "testprobepingpool"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.#:                         "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1633598275.failover_delay: "" => "0"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1633598275.host:           "" => "10.3.0.1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1633598275.priority:       "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1633598275.run_probes:     "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1633598275.state:          "" => "NORMAL"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1633598275.threshold:      "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.1633598275.weight:         "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4166511353.failover_delay: "" => "0"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4166511353.host:           "" => "10.3.0.2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4166511353.priority:       "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4166511353.run_probes:     "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4166511353.state:          "" => "NORMAL"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4166511353.threshold:      "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.4166511353.weight:         "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   run_probes:                      "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   ttl:                             "" => "30"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   zone:                            "" => "1keufrf3.test"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158: ultradns_tcpool.it: Creating...
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   act_on_probes:                   "" => "false"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   backup_record_failover_delay:    "" => "30"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   backup_record_rdata:             "" => "10.6.1.4"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   description:                     "" => "traffic controller pool with all settings tuned"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   hostname:                        "" => "<computed>"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   max_to_lb:                       "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   name:                            "" => "1keufrf3.test"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.#:                         "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2826722820.failover_delay: "" => "30"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2826722820.host:           "" => "10.6.1.1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2826722820.priority:       "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2826722820.run_probes:     "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2826722820.state:          "" => "ACTIVE"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2826722820.threshold:      "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   rdata.2826722820.weight:         "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   run_probes:                      "" => "false"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   ttl:                             "" => "300"
TestUltraDNSPlugin 2020-04-27T16:47:08+05:30 command.go:158:   zone:                            "" => "1keufrf3.test"
TestUltraDNSPlugin 2020-04-27T16:47:12+05:30 command.go:158: ultradns_record.foobar: Creation complete after 5s (ID: terraform.1keufrf3.test.)
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158: ultradns_record.it: Creation complete after 4s (ID: test-record-txt.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158: ultradns_rdpool.it: Creation complete after 4s (ID: test-rdpool-minimal.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158: ultradns_dirpool.test-dirpool-minimal: Creation complete after 4s (ID: testdirpoolminimal.com.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-maximal: Creation complete after 5s (ID: testprobehttpmaximal.com.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158: ultradns_probe_http.test-probe-http-maximal: Creating...
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   agents.#:                                             "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   agents.2144410488:                                    "" => "AMSTERDAM"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   agents.4091180299:                                    "" => "DALLAS"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_id:                                              "" => "<computed>"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.#:                                         "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.total_limits.#:                          "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.total_limits.0.critical:                 "" => "14"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.total_limits.0.fail:                     "" => "15"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.total_limits.0.warning:                  "" => "13"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.#:                           "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.follow_redirects:          "" => "true"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.#:                   "" => "4"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1349952704.critical: "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1349952704.fail:     "" => "3"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1349952704.name:     "" => "run"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1349952704.warning:  "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1959786783.critical: "" => "11"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1959786783.fail:     "" => "12"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1959786783.name:     "" => "connect"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1959786783.warning:  "" => "10"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.2720402232.critical: "" => "5"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.2720402232.fail:     "" => "6"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.2720402232.name:     "" => "avgConnect"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.2720402232.warning:  "" => "4"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.896769211.critical:  "" => "8"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.896769211.fail:      "" => "9"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.896769211.name:      "" => "avgRun"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.896769211.warning:   "" => "7"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.method:                    "" => "POST"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.transmitted_data:          "" => "{}"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.url:                       "" => "http://www.google.com"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   interval:                                             "" => "ONE_MINUTE"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   name:                                                 "" => "testprobehttpmaximal.com"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   pool_record:                                          "" => "10.2.1.1"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   threshold:                                            "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   zone:                                                 "" => "1keufrf3.test"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158: ultradns_tcpool.it: Creation complete after 4s (ID: 1keufrf3.test.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158: ultradns_tcpool.test-probe-ping-pool: Creation complete after 4s (ID: testprobepingpool.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158: ultradns_probe_ping.it: Creating...
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   agents.#:                               "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   agents.0:                               "" => "DALLAS"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   agents.1:                               "" => "AMSTERDAM"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   interval:                               "" => "ONE_MINUTE"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   name:                                   "" => "testprobepingpool"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   ping_id:                                "" => "<computed>"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   ping_probe.#:                           "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   ping_probe.0.limit.#:                   "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   ping_probe.0.limit.3257917790.critical: "" => "3"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   ping_probe.0.limit.3257917790.fail:     "" => "4"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   ping_probe.0.limit.3257917790.name:     "" => "total"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   ping_probe.0.limit.3257917790.warning:  "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   ping_probe.0.limit.3375621462.critical: "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   ping_probe.0.limit.3375621462.fail:     "" => "3"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   ping_probe.0.limit.3375621462.name:     "" => "lossPercent"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   ping_probe.0.limit.3375621462.warning:  "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   ping_probe.0.packet_size:               "" => "56"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   ping_probe.0.packets:                   "" => "15"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   threshold:                              "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   zone:                                   "" => "1keufrf3.test"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158: ultradns_dirpool.test-dirpool-maximal: Creation complete after 5s (ID: testdirpoolmaximal.com.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-minimal: Creation complete after 5s (ID: testprobehttpminimal.com.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158: ultradns_probe_http.test-probe-http-minimal: Creating...
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   agents.#:                                             "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   agents.2144410488:                                    "" => "AMSTERDAM"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   agents.4091180299:                                    "" => "DALLAS"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_id:                                              "" => "<computed>"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.#:                                         "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.#:                           "" => "1"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.follow_redirects:          "" => "false"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.#:                   "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1349952704.critical: "" => "60"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1349952704.fail:     "" => "60"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1349952704.name:     "" => "run"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1349952704.warning:  "" => "60"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1959786783.critical: "" => "20"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1959786783.fail:     "" => "20"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1959786783.name:     "" => "connect"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.limit.1959786783.warning:  "" => "20"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.method:                    "" => "GET"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   http_probe.0.transaction.0.url:                       "" => "http://www.google.com"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   interval:                                             "" => "ONE_MINUTE"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   name:                                                 "" => "testprobehttpminimal.com"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   pool_record:                                          "" => "10.2.0.1"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   threshold:                                            "" => "2"
TestUltraDNSPlugin 2020-04-27T16:47:13+05:30 command.go:158:   zone:                                                 "" => "1keufrf3.test"
TestUltraDNSPlugin 2020-04-27T16:47:14+05:30 command.go:158: ultradns_probe_http.test-probe-http-maximal: Creation complete after 1s (ID: 060847CB4D0E41F3)
TestUltraDNSPlugin 2020-04-27T16:47:14+05:30 command.go:158: ultradns_probe_ping.it: Creation complete after 1s (ID: 060847CB4D0E41F7)
TestUltraDNSPlugin 2020-04-27T16:47:14+05:30 command.go:158: ultradns_probe_http.test-probe-http-minimal: Creation complete after 1s (ID: 060847CB4D0E41FB)
TestUltraDNSPlugin 2020-04-27T16:47:15+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:47:15+05:30 command.go:158: Apply complete! Resources: 12 added, 0 changed, 0 destroyed.
TestUltraDNSPlugin 2020-04-27T16:47:15+05:30 retry.go:72: terraform [destroy -input=false -force -no-color]
TestUltraDNSPlugin 2020-04-27T16:47:15+05:30 command.go:87: Running command terraform with args [destroy -input=false -force -no-color]
TestUltraDNSPlugin 2020-04-27T16:47:15+05:30 command.go:158: ultradns_tcpool.test-probe-ping-pool: Refreshing state... (ID: testprobepingpool.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:15+05:30 command.go:158: ultradns_record.it: Refreshing state... (ID: test-record-txt.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:15+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-minimal: Refreshing state... (ID: testprobehttpminimal.com.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:15+05:30 command.go:158: ultradns_tcpool.it: Refreshing state... (ID: 1keufrf3.test.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:15+05:30 command.go:158: ultradns_dirpool.test-dirpool-maximal: Refreshing state... (ID: testdirpoolmaximal.com.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:15+05:30 command.go:158: ultradns_record.foobar: Refreshing state... (ID: terraform.1keufrf3.test.)
TestUltraDNSPlugin 2020-04-27T16:47:15+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-maximal: Refreshing state... (ID: testprobehttpmaximal.com.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:15+05:30 command.go:158: ultradns_rdpool.it: Refreshing state... (ID: test-rdpool-minimal.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:15+05:30 command.go:158: ultradns_dirpool.test-dirpool-minimal: Refreshing state... (ID: testdirpoolminimal.com.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:17+05:30 command.go:158: ultradns_probe_ping.it: Refreshing state... (ID: 060847CB4D0E41F7)
TestUltraDNSPlugin 2020-04-27T16:47:18+05:30 command.go:158: ultradns_probe_http.test-probe-http-minimal: Refreshing state... (ID: 060847CB4D0E41FB)
TestUltraDNSPlugin 2020-04-27T16:47:19+05:30 command.go:158: ultradns_probe_http.test-probe-http-maximal: Refreshing state... (ID: 060847CB4D0E41F3)
TestUltraDNSPlugin 2020-04-27T16:47:19+05:30 command.go:158: ultradns_dirpool.test-dirpool-maximal: Destroying... (ID: testdirpoolmaximal.com.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:19+05:30 command.go:158: ultradns_probe_http.test-probe-http-minimal: Destroying... (ID: 060847CB4D0E41FB)
TestUltraDNSPlugin 2020-04-27T16:47:19+05:30 command.go:158: ultradns_rdpool.it: Destroying... (ID: test-rdpool-minimal.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:19+05:30 command.go:158: ultradns_record.it: Destroying... (ID: test-record-txt.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:19+05:30 command.go:158: ultradns_dirpool.test-dirpool-minimal: Destroying... (ID: testdirpoolminimal.com.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:19+05:30 command.go:158: ultradns_tcpool.it: Destroying... (ID: 1keufrf3.test.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:19+05:30 command.go:158: ultradns_probe_http.test-probe-http-maximal: Destroying... (ID: 060847CB4D0E41F3)
TestUltraDNSPlugin 2020-04-27T16:47:19+05:30 command.go:158: ultradns_probe_ping.it: Destroying... (ID: 060847CB4D0E41F7)
TestUltraDNSPlugin 2020-04-27T16:47:19+05:30 command.go:158: ultradns_record.foobar: Destroying... (ID: terraform.1keufrf3.test.)
TestUltraDNSPlugin 2020-04-27T16:47:20+05:30 command.go:158: ultradns_probe_http.test-probe-http-minimal: Destruction complete after 1s
TestUltraDNSPlugin 2020-04-27T16:47:20+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-minimal: Destroying... (ID: testprobehttpminimal.com.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:20+05:30 command.go:158: ultradns_dirpool.test-dirpool-maximal: Destruction complete after 1s
TestUltraDNSPlugin 2020-04-27T16:47:21+05:30 command.go:158: ultradns_rdpool.it: Destruction complete after 1s
TestUltraDNSPlugin 2020-04-27T16:47:21+05:30 command.go:158: ultradns_record.it: Destruction complete after 1s
TestUltraDNSPlugin 2020-04-27T16:47:21+05:30 command.go:158: ultradns_dirpool.test-dirpool-minimal: Destruction complete after 2s
TestUltraDNSPlugin 2020-04-27T16:47:21+05:30 command.go:158: ultradns_probe_ping.it: Destruction complete after 2s
TestUltraDNSPlugin 2020-04-27T16:47:21+05:30 command.go:158: ultradns_tcpool.test-probe-ping-pool: Destroying... (ID: testprobepingpool.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:21+05:30 command.go:158: ultradns_probe_http.test-probe-http-maximal: Destruction complete after 2s
TestUltraDNSPlugin 2020-04-27T16:47:21+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-maximal: Destroying... (ID: testprobehttpmaximal.com.1keufrf3.test)
TestUltraDNSPlugin 2020-04-27T16:47:21+05:30 command.go:158: ultradns_record.foobar: Destruction complete after 2s
TestUltraDNSPlugin 2020-04-27T16:47:22+05:30 command.go:158: ultradns_tcpool.it: Destruction complete after 2s
TestUltraDNSPlugin 2020-04-27T16:47:22+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-minimal: Destruction complete after 1s
TestUltraDNSPlugin 2020-04-27T16:47:22+05:30 command.go:158: ultradns_tcpool.test-tcp-pool-maximal: Destruction complete after 1s
TestUltraDNSPlugin 2020-04-27T16:47:22+05:30 command.go:158: ultradns_tcpool.test-probe-ping-pool: Destruction complete after 1s
TestUltraDNSPlugin 2020-04-27T16:47:22+05:30 command.go:158:
TestUltraDNSPlugin 2020-04-27T16:47:22+05:30 command.go:158: Destroy complete! Resources: 12 destroyed.
--- PASS: TestUltraDNSPlugin (14.77s)
PASS
ok      command-line-arguments  14.770s